LayoutTests/fast/table/inline-table-margin-baseline-expected.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Firefox is failing this test in the vertical case, so it would be good to make it a WPT test.

Reply:
Created WPT PR: https://github.com/web-platform-tests/wpt/pull/36337